src/decirest_handler_lib.erl

```diff
@@ -74,7 +75,7 @@ content_types_accepted_default(Req, State) ->
 
 is_exported(Module, Function, ArityList) when is_list(ArityList) ->
```

Comment:
this isn't new for this PR but Iv'e missed this function before, awesome idea. what do you think about returning the arity instead
```
is_exported(M, F, ALst) when is_list(ALst) ->
  ExportedArity = [A || A <- ALst, is_exported(M, F, A)],
  case ExportedArity of
    [] -> false,
    EA -> max(EA)
  end;
...
```
that would give us the possibility to do one less case in other places like

```
case is_exported(Module, Function, [2, 3]) of
  2 -> Module:function(A, B);
  3 -> Module:function(A, B, C);
  false -> default_function(A, B, C, D)
end,
...
```
we could probably make a default function that does this for the standard case of Payload, Req, State if we haven't already

Reply:
Should we allow multiple callbacks. I was thinking we should crash when user creates dead callbacks.

Reply:
Yes we could still return arity to make the code easier